src/lib/DataTable/DataTableRouted.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```

Comment:
@000-ooo-000 

could you add something similar here to allow 230. 

Thanks

Reply:
Modified rule